onnxruntime/core/optimizer/nhwc_transformer.cc

```diff
@@ -237,6 +249,35 @@ void NhwcTransformerImpl::TransformQLinearGlobalAveragePool(Node& node) {
   CreateNhwcArgument(node, node, nhwc_input->rank_);
 }
 
+void NhwcTransformerImpl::TransformSplit(Node& node) {
+  auto& input_defs = node.MutableInputDefs();
+
+  auto* nhwc_input = LookupNhwcArgument(input_defs[0]);
+  if (nhwc_input == nullptr) {
+    return;
+  }
+
+  // Change the axis attribute accordingly for NCHW to NHWC model.
+  const auto* axis_attr = graph_utils::GetNodeAttribute(node, "axis");
+  if (axis_attr != nullptr && utils::HasInt(*axis_attr)) {
+    int64_t axis = axis_attr->i();
+    if (axis < -nhwc_input->rank_ || axis >= nhwc_input->rank_) {
+      // direct return on invalid axis
+      return;
+    }
+    if (axis >= 1) {
+      axis = (axis == 1LL) ? (nhwc_input->rank_ - 1) : (axis - 1);
+    } else if (axis > - nhwc_input->rank_ && axis < 0) {
+      axis = (axis == -(nhwc_input->rank_ - 1)) ? (- 1) : (axis - 1);
+    }
+    node.AddAttribute("axis", axis);
+  }
+
+  input_defs[0] = nhwc_input->nhwc_arg_;
+  nhwc_input->remaining_original_uses_--;
+  CreateNhwcArgument(node, node, nhwc_input->rank_);
```

Comment:
nit: add space here to match other transform routines

Reply:
added